src/beacons/beacons.ts

```diff
@@ -0,0 +1,15 @@
+import { Results } from 'groupby-api';
+import { SendableOrigin } from 'gb-tracker-client/models';
+
+/** The event to trigger a search beacon. */
+export const TRACKER_SEARCH: string = 'gbe::tracker::search';
```

Comment:
I would suggest changing "tracker" to "beacon". "Beacon" is a generic enough term. It also means that another level won't be necessary since "search" would describe "beacon" rather than being under the "tracker" hierarchy, if that makes sense.

```suggestion
export const BEACON_SEARCH: string = 'gbe::beacon_search';
```

Reply:
I admit I don't understand the second part, but I'm happy to use `beacon` more generally instead of `tracker`.

Reply:
Having the single level makes it more consistent with all the other event names that we have in this repo. We're currently using the second level for other context-specific data like the group in the case of the cache.